### DISCUSSION TITLE
Change GTK+ Theme to Dark when changing theme at DDE Control Center

### DISCUSSION THREAD

**Discussion by eduardomozart** [linuxdeepin/developer-center#3348] [Features Request & Ideas | 特性请求 & 头脑风暴]
After changing the Deeping 20.6 theme to Dark Theme, GTK applications (like the ones downloaded at Flatpak) still uses the Light Theme. I had to donwload the ``gnome-tweak-tool`` package to change it to "Deepin-dark" at "Appearance" screen. The DDE Control Center should also change it automatically when the Dark Theme is enabled.

**Comment by BLumia** [comment]
It should work for regular GTK app but seems currently not for flatpak packages.

@meiyixiang 需求：切换亮色与暗色主题后，希望从 flatpak 获取的 GTK 应用也可以跟随切换主题。
